lib/Remotery.c

```diff
@@ -2366,12 +2373,19 @@ typedef struct
 
     rmtU32 frame_bytes_remaining;
     rmtU32 mask_offset;
+#ifdef _MSC_VER
+    pragma warning(push)
 
+    #pragma warning(disable:4201) // Nameless struct/union
+#endif
     union
     {
         rmtU8 data_mask[4];
         rmtU32 data_mask_u32;
     };
+#ifdef _MSC_VER
+    #pragma warning(pop)
+#endif
 } WebSocket;
 
```

Comment:
Not without breaking things.  For example if I name it "u" then you have to reference data_mask via u.